front/src/modules/timesStops/helpers/utils.ts

```diff
@@ -196,7 +196,11 @@ export function updateRowTimesAndMargin(
       newRowData.stopFor = undefined;
     }
   }
-  if (!newRowData.stopFor && op.fromRowIndex !== allWaypointsLength - 1) {
+  if (
+    !newRowData.stopFor &&
+    newRowData.onStopSignal &&
```

Comment:
Nit: does this change really matter? If we reset the field to false when it's already false, it's a no-op so should be fine as well?

(In general, I prefer keeping the list of conditions in a `if` as small as possible.)

Reply:
It does because we also set null/undefined to false, which means we need to add the row to the path to remember it. 

We could also put it in the format function, but here I think it makes more sense to directly avoid doing a modification rather than do it then drop it after the comparison

Reply:
Makes sense!